app/assets/stylesheets/components/_alerts.scss

```diff
@@ -0,0 +1,34 @@
+.alert {
+  border-radius: 10px;
+  color: $black;
+  margin-bottom: 10px;
+  padding: 15px;
+
+  &.success {
+    background: $white;
+    border: 1px solid $green;
+  }
+
+  &.notice {
+    background: $white;
+    border: 1px solid $blue;
+  }
+
+  &.warning {
+    background: $white;
+    border: 1px solid $yellow;
+  }
+
+  &.danger {
+    background: $white;
+    border: 1px solid $red;
+  }
+}
+
```

Comment:
@wontoan Much better! There's just a couple nit-picky things that you can address but won't really make too much difference.

First, you have `background: $white` in all four of the alert states.  Since i applies to all of them,  you can just move it outside and apply it once to the main `.alert` class.

Second, you're also repeating `border: 1px solid …`.  This is more stylistic than anything though does have some implications in CSS specificity.  More importantly, there is no base style.  So, if someone forgets to add one of the state classes, the alert won't be appropriately styled.  So, I recommend adding `border: 1px solid $default-color` to the base style and then just use `border-color: $color` to change the color in each of the variations.